framework/src/base/FEProblemBase.C

```diff
@@ -4364,7 +4364,7 @@ FEProblemBase::checkDependMaterialsHelper(const std::map<SubdomainID, std::vecto
         }
 
         oss << "\nThis will result in ambiguous material property calculations and lead to incorrect results.\n";
```

Comment:
Could you remove this line of the message?  It doesn't make much sense since we are now disallowing duplicate props altogether.